.gitattributes

```diff
@@ -1,16 +1,18 @@
-*.jpg binary
-*.png binary
-*.sln binary
-*.vcproj binary
-*.exe binary
-*.lda binary
-*.bin binary
-*.imd binary
-*.rim binary
-*.tap binary
-*.dsk binary
-*.vhd binary
-*.crd binary
-*.dck binary
+*.jpg -text
+*.png -text
+*.sln -text
+*.vcproj -text
+*.exe -text
+*.lda -text
+*.bin -text
+*.imd -text
+*.rim -text
+*.tap -text
+*.dsk -text
+*.vhd -text
+*.crd -text
+*.dck -text
 sim_rev.h export-subst
-
+*.c text

```

Comment:
As discussed: please leave .gitattributes unchanged.  I assume it is not actually necessary to have .cmake files be text (LF-only on Unix), right?  If it is, just add the *.cmake entry but don't touch the rest.

Reply:
Reverted to master's baseline. Also converted sim_disk.c, sim_fio.[ch] back to DOS-format line endings to reduce diffs.